src/Faker/Provider/hu_HU/Person.php

```diff
@@ -28,11 +27,17 @@ class Person extends \Faker\Provider\Person
         'Benedek', 'Hunor', 'Jenő', 'János', 'Mihály', 'Mátyás', 'Szervác', 'Zsombor', 'Zétény', 'Árpád',
     ];
 
-    protected static $firstNameMaleNe = [
-        'Albertné', 'Attiláné', 'Balázsné', 'Bencéné', 'Botondné', 'Doriánné', 'Endrené', 'Ernőné', 'Gáborné', 'Gézané', 'Imréné', 'Istvánné',
-        'Kevinné', 'Kornélné', 'Kristófné', 'Lászlóné', 'Milánné', 'Noelné', 'Olivérné', 'Ottóné', 'Patrikné', 'Péterné', 'Richárdné', 'Rudolfné',
-        'Sándorné', 'Vilmosné', 'Vincéné', 'Zoltánné', 'Zsoltné', 'Ádámné', 'Árminné', 'Áronné', 'Antalné', 'Barnáné', 'Barnabásné', 'Bendegúz',
-        'Benedekné', 'Hunorné', 'Jenőné', 'Jánosné', 'Mihályné', 'Mátyásné', 'Szervácné', 'Zsomborné', 'Zétényné', 'Árpádné',
+    protected static $lastNameFemaleMarried = [
```

Comment:
Could you please provide a source for these names.

Reply:
It is come from $lastName + "né" string. Let me explain this: If a girl has a name like "Antal Adél" if she married he can pick up his husbund's lastName (Bakos Sándor) with "né" tag. So she became Bakosné Antal Adél.